homalg/gap/HomalgFunctor.gd

```diff
@@ -156,10 +156,7 @@ DeclareAttribute( "OperationOfFunctor",
 ##  <#/GAPDoc>
 ##
 DeclareAttribute( "Genesis",
-        IsHomalgFunctor );
-
-DeclareAttribute( "IsIdentityOnObjects",
```

Comment:
These three lines were probably deleted by accident, right?

Reply:
@mohamed-barakat No, this is intentional. See https://github.com/homalg-project/homalg_project/pull/286/commits/6f1c79fd09732a98faaf1ee7cc9e469c13d30935 (this PR has several commits, each has additional explanations.